README.md

```diff
@@ -55,7 +56,7 @@ packages.
 
  * Save and exit.
 
- * Run `make`
+ * Run `make` (for a real benchmarking setup do *not* use `-j`)
```

Comment:
But where do we end with these instructions? We could go on and on and on and on. Anyone who does *something other than we explicitly told them* gets what they deserve.